LoopKit/GlucoseKit/GlucoseStore.swift

```diff
@@ -560,9 +560,18 @@ extension GlucoseStore {
     ///   - effects: An array of velocities describing the change in glucose samples compared to the specified effects
     public func getCounteractionEffects(start: Date, end: Date? = nil, to effects: [GlucoseEffect], _ completion: @escaping (_ effects: [GlucoseEffectVelocity]) -> Void) {
         getCachedGlucoseSamples(start: start, end: end) { (samples) in
-            completion(samples.counteractionEffects(to: effects))
+            completion(self.counteractionEffects(for: samples, to: effects))
         }
     }
+
+    /// Calculates a timeline of effect velocity (glucose/time) observed in glucose that counteract the specified effects.
+    ///
+    /// - Parameter:
+    ///   - samples: The observed timeline of samples
+    ///   - effects: An array of velocities describing the change in glucose samples compared to the specified effects
+    public func counteractionEffects<Sample: GlucoseSampleValue>(for samples: [Sample], to effects: [GlucoseEffect]) -> [GlucoseEffectVelocity] {
+        samples.counteractionEffects(to: effects)
```

Comment:
Not sure this wrapper is adding much?

Reply:
It's really just to keep counteraction effect computation part of GlucoseStore's API. This change creates parity with the `glucoseEffects` API in CarbStore—although that one does require some details of settings in order to compute.